docs/ProfilerOptions.md

```diff
@@ -0,0 +1,113 @@
+| `--lock DURATION`  | `lock=DURATION`   | In lock profiling mode, sample contended locks when total lock duration overflows the threshold                                                                                                                                                                                                                                                                                                                  |
+| `-j N`             | `jstackdepth=N`   | Sets the maximum stack depth. The default is 2048.</br>Example: `asprof -j 30 8983`                                                                                                                                                                                                                                                                                                                              |
+| `-I include`       | `include=PATTERN` | Filter stack traces by the given pattern(s). `-I` defines the name pattern that *must* be present in the stack traces. `-I` can be specified multiple times. A pattern may begin or end with a star `*` that denotes any (possibly empty) sequence of characters.</br>Example: `asprof -I 'Primes.*' -I 'java/*' 8983`                                                                                           |
+| `-X exclude`       | `exclude=PATTERN` | Filter stack traces by the given pattern(s). `-X` defines the name pattern that *must not* occur in any of stack traces in the output. `-X` can be specified multiple times. A pattern may begin or end with a star `*` that denotes any (possibly empty) sequence of characters.</br>Example: `asprof -X '*Unsafe.park*' 8983`                                                                                  |
+| `-L level`         | `loglevel=level`  | Log level: `debug`, `info`, `warn`, `error` or `none`.                                                                                                                                                                                                                                                                                                                                                           |
+| `-F features`      | `features=LIST`   | Comma separated list of HotSpot-specific features to include in stack traces. Supported features are:<ul><li>`vtable` - display targets of megamorphic virtual calls as an extra frame on top of `vtable stub` or `itable stub`.</li><li>`comptask` - display current compilation task (a Java method being compiled) in a JIT compiler stack trace.</li><li>`pcaddr` - display instruction addresses.</li></ul> |
```

Comment:
In the agent parameters, features should separated by another delimiter (e.g. `+`), because `,` is already used as a delimiter for different options.

Reply:
Also, for each feature it'll be useful to add a link to the corresponding GitHub issue.

Reply:
Addressed in [6526389](https://github.com/async-profiler/async-profiler/pull/1029/commits/65263898accd8b0d26d286d63e4f1ef990ad9fbf)